tools/run_tests/performance/loadtest_config.py

```diff
@@ -65,9 +65,6 @@ def now_string() -> str:
 
 def validate_loadtest_name(name: str) -> None:
     """Validates that a LoadTest name is in the expected format."""
-    if len(name) > 63:
```

Comment:
I would keep the test, replacing the value 63 with 253 (in case someone specifies a really long prefix).

See https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names.

Reply:
Make sense. Done.